smurfs/src/reducers/index.js

```diff
@@ -0,0 +1,63 @@
+import {FETCH_SMURFS,
+        FETCH_SMURFS_SUCCESS,
+        FETCH_SMURFS_ERROR,
+        ADD_SMURF,
+        ADD_SMURF_SUCCESS,
+        ADD_SMURF_ERROR} from "../actions/index"
+
+const initialState = {
+  smurfs:[],
+  isLoading: false,
+  error: ''
+};
+
+export function reducer (state = initialState, action) {
```

Comment:
Since you have 2 action creator functions, conventionally you should also have 2 reducers and then combine them using a combineReducer function